### DISCUSSION TITLE
Failed to update dependencies

### DISCUSSION THREAD

**Discussion by ephraimberkovitch** [deta/cloud-docs#341] [Help] [answered]
Hi, my dependencies are failing to be installed in your micro. Is it because the size limits being exceeded? At the startup, I also need to install the BERT model. Probably, even if the dependencies would succeed to install, the starup would fail due to the large size of the model?
And another question - how can I configure to use Python 3.8 instead of Python 3.9, which to be the default Python runtime?

anyio==3.6.1
certifi==2022.6.15
charset-normalizer==2.1.0
click==8.1.3
cycler==0.11.0
descartes==1.1.0
fastapi==0.78.0
filelock==3.7.1
fonttools==4.34.4
h11==0.13.0
httptools==0.4.0
huggingface-hub==0.8.1
idna==3.3
joblib==1.1.0
kiwisolver==1.4.4
matplotlib==3.5.2
numpy==1.23.1
packaging==21.3
pandas==1.4.3
Pillow==9.2.0
pydantic==1.9.1
pyparsing==3.0.9
pyplutchik==0.0.7
python-dateutil==2.8.2
python-dotenv==0.20.0
python-multipart==0.0.5
pytz==2022.1
PyYAML==6.0
regex==2022.7.9
requests==2.28.1
sacremoses==0.0.53
Shapely==1.8.2
six==1.16.0
sniffio==1.2.0
starlette==0.19.1
tokenizers==0.10.3
torch==1.12.0
tqdm==4.64.0
transformers==4.14.1
typing-extensions==4.3.0
urllib3==1.26.10
uvicorn==0.18.2
uvloop==0.16.0
watchfiles==0.15.0
websockets==10.3

**Comment by lemonyte** [comment, marked as answer]
Pytorch is causing the error because it exceeds the 250mb limit. To update your Micro's runtime run `deta update --runtime python3.8` as per [the docs](https://docs.deta.sh/docs/cli/commands/#deta-update).
